spec/facility_spec.rb

```diff
@@ -2,25 +2,199 @@
+      registrant_1 = Registrant.new('Bruce', 18, true)
+      registrant_2 = Registrant.new('Timmy', 16, true)
+      
+      @facility_1.add_service('Road Test')
+      @facility_1.add_service('Written Test')
+      @facility_1.administer_written_test(registrant_1)
+      @facility_1.administer_road_test(registrant_1)
+      
+      expect(@facility_1.administer_road_test(registrant_2)).to eq(false)
+      expect(registrant_1.license_data).to eq({:written=>true, :license=>true, :renewed=>false})
+      expect(registrant_2.license_data).to eq({:written=>false, :license=>false, :renewed=>false})
+    end
+  end
+  
+  describe '#renew_drivers_license' do
+    it 'can check if service offered' do
+      registrant_1 = Registrant.new('Bruce', 18, true)
+      
+      expect(@facility_1.renew_drivers_license(registrant_1)).to eq(false)
+      expect(@facility_2.renew_drivers_license(registrant_1)).to eq(false)
+      
+      @facility_1.add_service('Road Test')
+      @facility_1.add_service('Written Test')
+      @facility_1.add_service('Renew License')
+
+      @facility_1.administer_written_test(registrant_1)
+      @facility_1.administer_written_test(registrant_1)
+      @facility_1.administer_road_test(registrant_1)
+        
+      expect(@facility_1.renew_drivers_license(registrant_1)).to eq(true)
+      expect(@facility_2.renew_drivers_license(registrant_1)).to eq(false)
+    end
+    it 'checks if :road_test == true' do
+      registrant_1 = Registrant.new('Bruce', 18, true)
+      registrant_2 = Registrant.new('Timmy', 16, true)
+      
+      @facility_1.add_service('Road Test')
+      @facility_1.add_service('Written Test')
+      @facility_1.add_service('Renew License')
+      @facility_1.administer_written_test(registrant_1)
+      @facility_1.administer_road_test(registrant_1)
+      @facility_1.administer_written_test(registrant_2)
+
+      
+      expect(@facility_1.renew_drivers_license(registrant_1)).to eq(true)
+      expect(@facility_1.renew_drivers_license(registrant_2)).to eq(false)
+    end
+    it 'sets Registrant @renewed == true if @license == true' do
+      registrant_1 = Registrant.new('Bruce', 18, true)
+      registrant_2 = Registrant.new('Timmy', 16, true)
+      
+      @facility_1.add_service('Road Test')
+      @facility_1.add_service('Written Test')
+      @facility_1.add_service('Renew License')
+
+      @facility_1.administer_written_test(registrant_1)
+      @facility_1.administer_road_test(registrant_1)
+      @facility_1.renew_drivers_license(registrant_1)
+      @facility_1.administer_written_test(registrant_2)
+      
+      expect(@facility_1.renew_drivers_license(registrant_1)).to eq(true)
+      expect(@facility_1.renew_drivers_license(registrant_2)).to eq(false)
+      expect(registrant_1.license_data).to eq({:written=>true, :license=>true, :renewed=>true})
+      expect(registrant_2.license_data).to eq({:written=>true, :license=>false, :renewed=>false})
     end
```

Comment:
I think that these tests look good. I noticed that you created helper methods of `add_plate` and `add_registration` are those tested in the `vehicle` class??